src/third-party/PhishingController.test.ts

```diff
@@ -108,6 +112,22 @@ describe('PhishingController', () => {
     expect(controller.test('xn--myetherallet-4k5fwn.com')).toBe(true);
   });
 
+  it('should return positive result for unsafe unicode domain from the PhishFort blacklist', async () => {
+    const controller = new PhishingController();
+    await controller.updatePhishingLists();
+    expect(controller.test('e4d600ab9141b7a9859511c77e63b9b3.com')).toBe(true);
+  });
+
+  it('should return negative result for unsafe unicode domain if the PhishFort blacklist returns 304', async () => {
+    nock('https://cdn.jsdelivr.net', { allowUnmocked: true })
```

Comment:
We are actively trying to eliminate network requests from our unit tests, so using `allowUnmocked` is not ideal. I see that this test suite is already mostly unmocked though. I can fix this in a follow-up PR.

Reply:
Noted. I don't know Jest well, would it be possible to load this from a set of abridged test configs that are part of the repo?

Reply:
I'm not _entirely_ sure what you mean, but it sounds like you're asking if we could setup a mock and reuse it on most tests, and the answer to that is definitely yes. Not sure whether Jest could help with that but that's something we can do with nock. We can create a "default" mock and call it in each test, and only mock manually in cases like this where we want to test some special response.

Reply:
Regarding this, sorry yeah, did mean Jest and/or Nock. Resolving this for now in that case.